src/PaperG/FirehoundBlob/BlobInterface.php

```diff
@@ -13,6 +13,4 @@ public function toArray();
      * @param $array array
      */
     public function fromArray($array);
-
-    public function validate();
```

Comment:
Aiming on using validators, so they won't be on the blob.  I'm pulling the BasicInfo validation off when we write the validators.